config.yaml

```diff
@@ -1 +1,15 @@
-options: {}
+options:
+  webserver_workers:
+    type: int
+    description: The number of gunicorn worker processes for handling requests.
```

Comment:
These configuration options were intended to be a little more generic, do you think it would be better not to mention gunicorn here?

Reply:
Similar comment for the other configuration options

Reply:
Updated, thanks!